### ISSUE TITLE
Build fails with recent EUMM and some make variants

### ISSUE THREAD

**Issue by eserte** [ajgb/crypt-curve25519#5]
On FreeBSD systems with BSD make and recent ExtUtils::MakeMaker (>=7.16) I see the following error while trying to test the distribution:
``
$ make test
"/usr/perl5.25.9p/bin/perl5.25.9" -MExtUtils::Command::MM -e 'cp_nonempty' -- Curve25519.bs blib/arch/auto/Crypt/Curve25519/Curve25519.bs 644
make: don't know how to make dynamic. Stop
*** [subdirs-test_dynamic] Error code 2

Stop in /usr/local/src/CPAN/build/2017022721/Crypt-Curve25519-0.05-0.
```

The problem is probably caused by a change in ExtUtils::MakeMaker and discussed in this ticket: https://rt.cpan.org/Ticket/Display.html?id=117800

**Comment by eserte** [comment]
A possible patch:
http://cpan.cpantesters.org/authors/id/S/SR/SREZIC/patches/Crypt-Curve25519-0.05-RT117800.patch

**Comment by ajgb** [comment]
Could you test just released version 0.06?

Thank you,
Alex

**Comment by eserte** [comment]
0.06 works fine.
